From a0e1ddf87544a8371fff110c451576016ca81fb3 Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@ubuntu.com>
Date: Sun, 30 Dec 2012 09:57:58 +0000
Subject: [PATCH 073/364] * grub-core/bus/usb/ehci.c (grub_ehci_pci_iter):
 Remove incorrect __attribute__ ((unused)). * grub-core/video/bochs.c
 (find_card): Likewise. * grub-core/video/cirrus.c (find_card): Likewise. *
 grub-core/video/radeon_fuloong2e.c (find_card): Likewise. *
 grub-core/video/sis315pro.c (find_card): Likewise. * grub-core/video/sm712.c
 (find_card): Likewise.

---
 ChangeLog                          | 10 ++++++++++
 grub-core/bus/usb/ehci.c           |  3 +--
 grub-core/video/bochs.c            |  2 +-
 grub-core/video/cirrus.c           |  2 +-
 grub-core/video/radeon_fuloong2e.c |  4 ++--
 grub-core/video/sis315pro.c        |  4 ++--
 grub-core/video/sm712.c            |  4 ++--
 7 files changed, 19 insertions(+), 10 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index 3bca6fd..36f1bff 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,13 @@
+2012-12-30  Colin Watson  <cjwatson@ubuntu.com>
+
+	* grub-core/bus/usb/ehci.c (grub_ehci_pci_iter): Remove incorrect
+	__attribute__ ((unused)).
+	* grub-core/video/bochs.c (find_card): Likewise.
+	* grub-core/video/cirrus.c (find_card): Likewise.
+	* grub-core/video/radeon_fuloong2e.c (find_card): Likewise.
+	* grub-core/video/sis315pro.c (find_card): Likewise.
+	* grub-core/video/sm712.c (find_card): Likewise.
+
 2012-12-28  Colin Watson  <cjwatson@ubuntu.com>
 
 	* util/grub-mkconfig.in: Accept GRUB_TERMINAL_OUTPUT=vga_text.
diff --git a/grub-core/bus/usb/ehci.c b/grub-core/bus/usb/ehci.c
index d99a4be..dc5bf71 100644
--- a/grub-core/bus/usb/ehci.c
+++ b/grub-core/bus/usb/ehci.c
@@ -455,8 +455,7 @@ grub_ehci_reset (struct grub_ehci *e)
 
 /* PCI iteration function... */
 static int NESTED_FUNC_ATTR
-grub_ehci_pci_iter (grub_pci_device_t dev,
-		    grub_pci_id_t pciid __attribute__ ((unused)))
+grub_ehci_pci_iter (grub_pci_device_t dev, grub_pci_id_t pciid)
 {
   grub_uint8_t release;
   grub_uint32_t class_code;
diff --git a/grub-core/video/bochs.c b/grub-core/video/bochs.c
index 79cae65..f6db137 100644
--- a/grub-core/video/bochs.c
+++ b/grub-core/video/bochs.c
@@ -210,7 +210,7 @@ grub_video_bochs_setup (unsigned int width, unsigned int height,
   int pitch, bytes_per_pixel;
   grub_size_t page_size;        /* The size of a page in bytes.  */
 
-  auto int NESTED_FUNC_ATTR find_card (grub_pci_device_t dev, grub_pci_id_t pciid __attribute__ ((unused)));
+  auto int NESTED_FUNC_ATTR find_card (grub_pci_device_t dev, grub_pci_id_t pciid);
   int NESTED_FUNC_ATTR find_card (grub_pci_device_t dev, grub_pci_id_t pciid)
     {
       grub_pci_address_t addr;
diff --git a/grub-core/video/cirrus.c b/grub-core/video/cirrus.c
index 7fad50e..e711119 100644
--- a/grub-core/video/cirrus.c
+++ b/grub-core/video/cirrus.c
@@ -245,7 +245,7 @@ grub_video_cirrus_setup (unsigned int width, unsigned int height,
   int found = 0;
   int pitch, bytes_per_pixel;
 
-  auto int NESTED_FUNC_ATTR find_card (grub_pci_device_t dev, grub_pci_id_t pciid __attribute__ ((unused)));
+  auto int NESTED_FUNC_ATTR find_card (grub_pci_device_t dev, grub_pci_id_t pciid);
   int NESTED_FUNC_ATTR find_card (grub_pci_device_t dev, grub_pci_id_t pciid)
     {
       grub_pci_address_t addr;
diff --git a/grub-core/video/radeon_fuloong2e.c b/grub-core/video/radeon_fuloong2e.c
index 32f66c7..45a68ed 100644
--- a/grub-core/video/radeon_fuloong2e.c
+++ b/grub-core/video/radeon_fuloong2e.c
@@ -69,8 +69,8 @@ grub_video_radeon_fuloong2e_setup (unsigned int width, unsigned int height,
   int found = 0;
 
 #ifndef TEST
-  auto int NESTED_FUNC_ATTR find_card (grub_pci_device_t dev, grub_pci_id_t pciid __attribute__ ((unused)));
-  int NESTED_FUNC_ATTR find_card (grub_pci_device_t dev, grub_pci_id_t pciid __attribute__ ((unused)))
+  auto int NESTED_FUNC_ATTR find_card (grub_pci_device_t dev, grub_pci_id_t pciid);
+  int NESTED_FUNC_ATTR find_card (grub_pci_device_t dev, grub_pci_id_t pciid)
     {
       grub_pci_address_t addr;
       grub_uint32_t class;
diff --git a/grub-core/video/sis315pro.c b/grub-core/video/sis315pro.c
index 5d06daa..d213877 100644
--- a/grub-core/video/sis315pro.c
+++ b/grub-core/video/sis315pro.c
@@ -99,8 +99,8 @@ grub_video_sis315pro_setup (unsigned int width, unsigned int height,
   unsigned i;
 
 #ifndef TEST
-  auto int NESTED_FUNC_ATTR find_card (grub_pci_device_t dev, grub_pci_id_t pciid __attribute__ ((unused)));
-  int NESTED_FUNC_ATTR find_card (grub_pci_device_t dev, grub_pci_id_t pciid __attribute__ ((unused)))
+  auto int NESTED_FUNC_ATTR find_card (grub_pci_device_t dev, grub_pci_id_t pciid);
+  int NESTED_FUNC_ATTR find_card (grub_pci_device_t dev, grub_pci_id_t pciid)
     {
       grub_pci_address_t addr;
       grub_uint32_t class;
diff --git a/grub-core/video/sm712.c b/grub-core/video/sm712.c
index 5f22c40..d780983 100644
--- a/grub-core/video/sm712.c
+++ b/grub-core/video/sm712.c
@@ -370,8 +370,8 @@ grub_video_sm712_setup (unsigned int width, unsigned int height,
   grub_err_t err;
   int found = 0;
 
-  auto int NESTED_FUNC_ATTR find_card (grub_pci_device_t dev, grub_pci_id_t pciid __attribute__ ((unused)));
-  int NESTED_FUNC_ATTR find_card (grub_pci_device_t dev, grub_pci_id_t pciid __attribute__ ((unused)))
+  auto int NESTED_FUNC_ATTR find_card (grub_pci_device_t dev, grub_pci_id_t pciid);
+  int NESTED_FUNC_ATTR find_card (grub_pci_device_t dev, grub_pci_id_t pciid)
     {
       grub_pci_address_t addr;
       grub_uint32_t class;
-- 
1.8.1.4

